tools/depends/target/flatbuffers/Makefile

```diff
@@ -31,12 +31,11 @@ CMAKE_OPTIONS := -DCMAKE_BUILD_TYPE=Release \
                  $(CMAKE_OPTIONS)
 BUILDDIR = $(PLATFORM)/build-cmake # 'build' conflicts with file BUILD on case-insensitive FS
 
-all: .installed-$(PLATFORM)
+include ../../download-files.include
```

Comment:
I think this could just be add to the original VERSION include line? Unless the above comment precludes that because of setting hash_tool, etc

Reply:
it works somewhat, but download paths are messed up. I prefer to keep it as is